src/components/Wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from "react";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+import Navigation from "../Navigation";
+import Footer from "../Footer";
+
+export default function Wrapper(props) {
+    const { me, children } = props;
+    return (
+        <Fragment>
+            <Navigation me={me} />
+            {!children ? (
+                <div style={{ textAlign: "center" }}>
```

Comment:
why style not in class

Reply:
I think for a single use there is no need to create a class in React.